newsletter 6 (#57)
Openscapes/website: content/blog/news-februrary-2023.md

---
title: "Openscapes Newsletter #6: Winter 2023"
author: Julie Lowndes and Stefanie Butland

> date: '2022-02-21'

Reviewer: ```suggestion
date: '2023-02-21'
```

slug: news-winter-2023
categories:
  - community
tags:
  - newsletter
  - nasa-framework
  - noaa-fisheries

> banner: img/blog/mrlowndes_carrizo_plain.png

Reviewer: Not a huge deal but this is actually Santa Barbara, maybe it could be titled `sb_paradise_road.png`

Author: changed

---

## **Openscapes Newsletter #6: Winter 2023**

*Welcome to Openscapes' sixth newsletter! If you're interested in seeing these infrequent updates in your inbox, please [sign up here](https://docs.google.com/forms/d/e/1FAIpQLSdgVXRp3V-w94GPWkR31RUfyBl37EphdQSlCOcnyeNlf8OLWw/viewform) (linked from our [get involved](https://openscapes.org/contact) page).*

------------------------------------------------------------------------

> Let's come back to the core of what we do: we engage, empower, and amplify. We connect and elevate, whether that is on tech like Quarto and JupyterHubs or communities like R-Ladies, Ladies of Landsat, Black in Marine Science, and NASA Earthdata. This isn't about "getting" people to do open science, it's about welcoming folks to better ways of working, keeping kindness always at the forefront.

Reviewer: ```suggestion
Hello! As we continue into 2023, we at Openscapes continue to come back to the core of what we do: we engage, empower, and amplify. Whether it is with tech like Quarto and JupyterHubs or communities like R-Ladies, Ladies of Landsat, Black in Marine Science, and NASA Earthdata, it's about welcoming folks to better ways of working and open science.
```

> We've got big problems like climate change, social justice, democracy, and the ongoing pandemic, and folks feel stuck on the "little" problems of copy-pasting, and moving files around.

Reviewer: ```suggestion
In 2023, we're committed to being braver in connecting the impact of our work to the environment and the climate movement; to culture and mindset change for individuals, for teams, and for institutions; and as always, to kinder science.
```

**In the last year**, what emerged was a clear sense of movement building towards better science for future us. The Openscapes community grew as Champions and Mentors prioritized time for learning, sharing, teaching with each other, and also advocated for themselves and their colleagues to have time to do so.

> -   [NASA Openscapes Mentors](https://nasa-openscapes.github.io/mentors.html) gave talks, participated and created in public forums, taught workshops, and built software infrastructure and teaching resources to help researchers migrate their workflows to the cloud.

Reviewer: ```suggestion
-   [NASA Openscapes Mentors](https://nasa-openscapes.github.io/mentors.html) gave talks, participated and created in public forums, taught workshops, and built software infrastructure and teaching resources to help colleagues and researchers migrate their workflows to the cloud.
```

> -   [NOAA Fisheries Mentors](https://nmfs-openscapes.github.io/mentors.html) supported their colleagues through six Champions Cohorts. Read about the experiences and impacts of Fall 2022 Openscapes training [nationwide](https://openscapes.org/blog/2023/01/24/2022-noaa-nmfs-fall/), with additional posts from NOAA [Alaska](https://openscapes.org/blog/2023/02/16/2022-noaa-afsc-fall/), and [Southwest](https://openscapes.org/blog/2022/12/09/case-study-swfsc-sael/), Summer [Southeast](https://openscapes.org/blog/2022/09/30/2022-sefsc-summer/), and Spring [Alaska](https://openscapes.org/blog/2022/04/07/afsc-supportive-forum/) Fisheries Science Centers. Additional resources include the [NMFS Openscapes](https://nmfs-openscapes.github.io/) site and [NMFS Open Science Resource Book](https://nmfs-opensci.github.io/ResourceBook/).

Reviewer: ```suggestion
-   [NOAA Fisheries Mentors](https://nmfs-openscapes.github.io/mentors.html) supported their colleagues through six Champions Cohorts. Read about the experiences and impacts of Fall 2022 Openscapes training [nationwide](https://openscapes.org/blog/2023/01/24/2022-noaa-nmfs-fall/), with additional posts from NOAA Fisheries [Alaska](https://openscapes.org/blog/2023/02/16/2022-noaa-afsc-fall/), and [Southwest](https://openscapes.org/blog/2022/12/09/case-study-swfsc-sael/), Summer [Southeast](https://openscapes.org/blog/2022/09/30/2022-sefsc-summer/), and Winter [Alaska](https://openscapes.org/blog/2022/04/07/afsc-supportive-forum/) Fisheries Science Centers. Additional resources include the [NMFS Openscapes](https://nmfs-openscapes.github.io/) site and [NMFS Open Science Resource Book](https://nmfs-opensci.github.io/ResourceBook/).
```

> -   CalEPA Mentors forked the Champions program and co-led the first Cohort for the California Water Boards ([blog post](https://www.openscapes.org/blog/2022/12/02/swrcb-2022/)) and fully documented it in a Quarto book, [Openscapes at the Water Boards](https://cawaterboarddatacenter.github.io/swrcb-openscapes/).

Reviewer: ```suggestion
-   [CalEPA Mentors](https://cawaterboarddatacenter.github.io/swrcb-openscapes/contact.html) forked the Champions program and co-led the first Cohort for the California Water Boards ([blog post](https://www.openscapes.org/blog/2022/12/02/swrcb-2022/)) and fully documented it in a Quarto book, [Openscapes at the Water Boards](https://cawaterboarddatacenter.github.io/swrcb-openscapes/).
```

-   240 Champions "graduated"! Their work and energy has been awesome! It's so wonderful to see teams of Champions come back to guest-teach their peers and share their ongoing journeys toward more open kinder science.

-   Ileana Fenwick and Stefanie Butland joined the Openscapes team ([tweet](https://twitter.com/openscapes/status/1534951248369025031))! They both do incredibly valuable work and we are so grateful to work together.

-   We hosted 3 Community Calls: A Qualitative Data Analysis Chat with Dr. Beth Duckles; Hello Quarto! A Quarto Chat with NASA Openscapes, co-hosted with R-Ladies Santa Barbara; Reimagining open science as part of the climate movement, a chat with Dr. Monica Granados ([summary posts](https://www.openscapes.org/tags/community-call/) of Community Calls).

> -   Julie gave a co-keynote at rstudio::conf(2022) with Mine Cetinkaya-Rundel, titled Hello Quarto: share, collaborate, teach, reimagine. This was the official launch of Quarto ([blog post](https://www.openscapes.org/blog/2022/08/10/quarto-keynote/)).

Reviewer: ```suggestion
-   Julie gave a co-keynote at rstudio::conf(2022) with Mine Cetinkaya-Rundel, titled Hello Quarto: share, collaborate, teach, reimagine. This was the official launch of Quarto ([blog post](https://www.openscapes.org/blog/2022/08/10/quarto-keynote/)) and it was such an honor to share about the Openscapes community on this big stage!
```

-   One of the things we hear often from researchers is that they feel like they can't take vacation - too much rests solely on their shoulders, solely on their laptops. In the shared-joy and sustainability-through-open-science departments, Julie wrote about how our open science process made it possible for her to unplug for a 2-week vacation ([blog post](https://www.openscapes.org/blog/2022/11/03/vacationing-with-open-science/)).

> -   Shoutout to all the Champions and Mentors who have shared their successes - big and small - with us. You keep us energized!

Reviewer: ```suggestion
-   Shoutout to all the Champions and Mentors and community who have shared their successes - big and small - with us. You keep us energized!
```

**This year**, we're continuing our work with NASA, NOAA, and the California Water Boards and have started new relationships and programs. We're excited to continue to grow the Open Science movement as part of the Year of Open Science ([Biden Harris administration announcement](https://www.whitehouse.gov/ostp/news-updates/2023/01/11/fact-sheet-biden-harris-administration-announces-new-actions-to-advance-open-and-equitable-research/)).

> -   Already we've launched [Pathways to Open Science](https://openscapes.github.io/pathways-to-open-science/) for Black environmental & marine researchers to build community for the future of data intensive science. 80 people attended the first session! Led by Ileana Fenwick in partnership with [Black in Marine Science (BIMS)](https://www.blackinmarinescience.org/) and [Black Women in Ecology, Evolution, and Marine Science (BWEEMS)](https://www.bweems.org/), we are committing to offer this annually.

Reviewer: ```suggestion
-   Already we've launched [Pathways to Open Science](https://openscapes.github.io/pathways-to-open-science/) for Black environmental & marine researchers, led by Ileana Fenwick in partnership with [Black in Marine Science (BIMS)](https://www.blackinmarinescience.org/) and [Black Women in Ecology, Evolution, and Marine Science (BWEEMS)](https://www.bweems.org/). 80 people attended the first session(!) and we are committing to offer this annually.
```

-   At the [ESIP Winter meeting](https://2023januaryesipmeeting.sched.com/event/1EwXx) in January we coordinated a panel discussion. Mentors from NASA, NOAA Fisheries, CalEPA, and Pathways to Open Science programs shared their stories of open science movement building through Openscapes. Look for a summary blog post in March!

-   February through May, five research teams from the U.S Environmental Protection Agency (EPA) Center for Environmental Measurement and Modeling (CEMM) are participating in an Openscapes Champions Cohort with help from new Mentors Gayle Hagler and Jeff Hollister.

-   Thirty Openscapes Mentors across all of our programs and the Openscapes core team are coming together in their own six-session Cohort to practice Mentoring with a Coach Approach, led by [Tara Robertson](https://tararobertson.ca/), professional coach and Openscapes advisor and collaborator.

**Coming up**

> -   Nominations open for our 2023 [NASA Champions Cohort](https://nasa-openscapes.github.io/champions) that will run April - June.

Reviewer: ```suggestion
-   Nominations open for our 2023 [NASA Champions Cohort](https://nasa-openscapes.github.io/champions) that will run April - June. Please spread the word for teams using NASA Earthdata and interested in migrating workflows to the Cloud.
```

-   In June we'll launch a brand new program, **Openscapes Reflections**! This will be a 3-week program, modeled after leadership coaching in a mostly self-paced format. It will offer structure and accountability for participants to reflect and plan on analytical and reporting workflows to improve daily habits and fuel institutional culture change.

> In 2023, **we're committed to being braver** in connecting the impact of our work to the environment and the climate movement; to culture and mindset change for individuals, for teams, and for institutions; and as always, to kinder science.

Reviewer: ```suggestion
Thanks for reading, and here's to kinder science and the year of open science!
```
